operator/pkg/controllers/etcd/pod.go

```diff
@@ -187,10 +169,23 @@ func podSpecFor(controlPlane *v1alpha1.ControlPlane) *v1.PodSpec {
 	}
 }
 
+func generateEtcdBootstrapCmd(controlPlane *v1alpha1.ControlPlane) string {
```

Comment:
Some naming nits:
- I'd drop the word `generate` and just noun based getters (e.g. foo.generateBar() -> foo.bar())
- You're stuttering with the word `etcd` (e.g. `etcd.generateEtcd...`). We're already contextualized to etcd in this package. 

I wonder if we should be attaching this behavior to the controller, rather than unowned static methods.
`controller.bootstrapCommand(controlPlane *v1alpha1.ControlPlane)`

I can see leaving this as a static, as long as this package remains tightly scoped and we don't leak these functions anywhere.